### PR TITLE
Use Python 3 syntax in README.

### PR DESCRIPTION
**Pull request by tobami**
- Change print statement usage to Python 3's function
- Remove explicit unicode literal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,14 +145,14 @@ You might want to get some information about the location that corresponds to a
...
->>> print repr(geocoder.description_for_number(ch_number, "de"))
-u'Z\\xfcrich'
->>> print repr(geocoder.description_for_number(ch_number, "en"))
-u'Zurich'
->>> print repr(geocoder.description_for_number(ch_number, "fr"))
-u'Zurich'
->>> print repr(geocoder.description_for_number(ch_number, "it"))
-u'Zurigo'
+>>> print(repr(geocoder.description_for_number(ch_number, "de")))
+'Z\\xfcrich'
```
Comment (daviddrysdale) [inline README.md]: I think this comes out as `'Zürich'` in Python 3.

**Review by daviddrysdale — APPROVED** [review]
Yeah, it's probably about time -- thanks for the update.

One tiny tweak.

File: README.md (hunk 1 above)
Comment (tobami) [reply README.md]: Right. I'll also remove the now unnecessary `print(repr(` clutter.

**Comment by daviddrysdale** [comment]
Thanks!

**Pull request merged**